src/lib.rs

```diff
@@ -110,6 +110,8 @@ mod rtt_printer {
     )
 ))]
 mod serial_jtag_printer {
+    use core::sync::atomic::{AtomicBool, Ordering};
```

Comment:
Lets make this portable atomic ;)

Reply:
it's just load/store and I didn't want to introduce a new dependency, but sure why not :)